### PR TITLE
Fixing taxonomy and agency calls

### PR DESCRIPTION
**Pull request by vijayyadav06**
This PR fixes the taxonomy and agency calls. We still need to figure out how to do joins.

### CODE REVIEW

File: api/controllers/services/ckan/agencyService.js [hunk 1]
```diff
@@ -7,11 +7,11 @@ class AgencyService extends AbstractService {
         this.agencyServiceResourceId = this.config.agency_service_resource;
         this.serviceTaxonomyResourceId = this.config.service_taxonomy_resource;
         this.agencyResourceId = this.config.agency_resource;
-        this.uri = `${this.host}/api/3/action/datastore_search_sql?sql=SELECT * FROM "${this.agencyResourceId}" ${this.tableName} `;
+        this.uri = `${this.host}/api/v1/organization/handson_central_ohio/dataset/${this.agencyResourceId}/query?_format=json`;
         this.joinQuery = `
-        INNER JOIN "${this.agencyServiceResourceId}" agency_service ON agency."AGENCY_ID" = agency_service."AGENCY_ID" 
-        INNER JOIN "${this.serviceTaxonomyResourceId}" service_taxonomy ON agency_service."AGENCY_ID" = service_taxonomy."AGENCY_ID" 
-        AND agency_service."LINE_NUMBER" = service_taxonomy."LINE_NUMBER"
+        &where= INNER JOIN "${this.agencyServiceResourceId}" agency_service ON agency."agency_id" = agency_service."agency_id"
```
Comment (mihiramin89) [inline api/controllers/services/ckan/agencyService.js]: can we do JOINs in the new API?

File: api/utils/request.js [hunk 2]
```diff
@@ -74,7 +74,7 @@ class RequestUtils {
      * @param {Function} callBack Callback function
      */
     handleRequest(uri, res, callBack) {
-    console.log(uri)
+        console.log('uri -',uri)
```
Comment (mihiramin89) [inline api/utils/request.js]: should this be removed?

File: api/controllers/services/ckan/agencyService.js (hunk 1 above)
Comment (vijayyadav06) [reply api/controllers/services/ckan/agencyService.js]: no this part of code is not used and will be replaced in my updated PR.

File: api/utils/request.js (hunk 2 above)
Comment (vijayyadav06) [reply api/utils/request.js]: yes

File: api/controllers/models/agency.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 class Agency {
     constructor(res) {
-        this.id = res.AGENCY_ID;
-        this.name = res.NAME;
-        this.locationId = res.LOCATION_ID;
-        this.taxonomyId = res.TAXON_ID;
+        this.id = String(res.agency_id)
+        this.name = res.name
+        this.locationId = String(res.location_id)
+        this.taxonomyId = res.taxon_id
```
Comment (mihiramin89) [inline api/controllers/models/agency.js]: should this be wrapped in a `String()` like the other IDs?

File: api/controllers/services/ckan/locationService.js [hunk 4]
```diff
@@ -8,40 +8,38 @@ class LocationCkanService extends AbstractService {
...
-            filter = getFilters(filter, `${this.tableName}."AGENCY_ID" IN (${req.query.agencyId})`);
-        }
-        requestBody = this.uri + this.query + requestBody;
-        const queryString = this.queryUtils.getQueryString(req, requestBody, filter, this.tableName);
-        this.requestUtils.getList(queryString, res, locationWithCoord(Location.get));
+            filter = getFilters(filter, `${this.tableName}."agency_id" IN (${req.query.agencyId})`);
+        }*/
+        //requestBody = this.uri + this.query + requestBody;
+        const queryString = this.uri + `${this.serviceLocationResourceId}`+'/query?_format=json'+ this.queryUtils.setDefaultFilters('columns = location_id, location_number, street_1, street_2, zip, name, phone_area_code, phone_number, phone_extension, handicap_access')
+        console.log(this.requestUtils.getList(queryString, res, locationWithCoord(Location.get)));
```
Comment (mihiramin89) [inline api/controllers/services/ckan/locationService.js]: should this be wrapped in a `console.log`?

**Comment by vijayyadav06** [comment]
obsolete, have another PR coming.

**Pull request closed without merging**